simpa/core/processing_components/monospectral/field_of_view_cropping.py

```diff
@@ -48,7 +48,7 @@ def run(self, device: DigitalDeviceTwinBase):
         self.logger.debug(f"FOV (mm): {field_of_view_mm}")
         _, _, _, xdim_start, xdim_end,  ydim_start, ydim_end, zdim_start, zdim_end = compute_image_dimensions(field_of_view_mm, self.global_settings[Tags.SPACING_MM], self.logger)
         field_of_view_voxels = [xdim_start, xdim_end, zdim_start, zdim_end, ydim_start, ydim_end] # change ordering
-        field_of_view_voxels = [int(dim) for dim in field_of_view_voxels] # cast to int
+        field_of_view_voxels = [round_away_from_zero(dim) for dim in field_of_view_voxels] # cast to int
```

Comment:
I'm not sure if we want to round here at all or if we just want to truncate/cast to int. The `compute_image_dimensions` function converts the FOV in mm into pixel indices. The first three returned elements are the x, y and z dimension widths as integer values. The other `{x,y,z}dim_{start,end}` values are floats. Within `compute_image_dimensions` these floats are adjusted for offsets resulting from the rounding of the width to an integer. Applying an additional rounding here would then maybe make the FOV slightly larger than wanted, right? 

I'm not sure what the best way to convert the floats from `compute_image_dimensions` into ints is. When rounding away from zero or truncating/casting to int then the difference between the resulting `dim_start` and `dim_end` values might not be the same as the `dim_width` anymore. To achieve this, flooring or ceiling could be used but then the image might not be symmetric anymore. Depending on which behavior we would like to have we could use one or the other.

Reply:
I corrected it :)